### ISSUE TITLE
Corrupted code

### ISSUE THREAD

**Issue by ihor-drachuk** [microsoft/vscode-copilot-release#2090]
Type: <b>Bug</b>

Corrupted code

Extension version: 0.21.2
VS Code version: Code 1.94.2 (384ff7382de624fb94dbaf6da11977bba1ecd427, 2024-10-09T16:08:44.566Z)
OS version: Windows_NT x64 10.0.22621
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|13th Gen Intel(R) Core(TM) i9-13980HX (12 x 2419)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: unavailable_off<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|16.00GB (7.30GB free)|
|Process Argv|C:\\workspace\\project --crash-reporter-id 6a5a53b6-3f37-4200-887e-7d7a44df403e|
|Screen Reader|no|
|VM|100%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805:30301674
binariesv615:30325510
vsaa593:30376534
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
2i9eh265:30646982
962ge761:30959799
pythongtdpath:30769146
pythonnoceb:30805159
asynctok:30898717
pythonmypyd1:30879173
2e7ec940:31000449
pythontbext0:30879054
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
9c06g630:31013171
dvdeprecation:31068756
dwnewjupyter:31046869
impr_priority:31102340
nativerepl2:31139839
refactort:31108082
pythonrstrctxt:31112756
wkspc-onlycs-t:31132770
wkspc-ranged-t:31151552
cf971741:31144450
iacca1:31156133
notype1:31157159
5fd0e150:31155592
dwcopilot:31164048
icondisabled:31158250

```

</details>

Here is how generated code looks:
![Image](https://github.com/user-attachments/assets/8361f2d2-e2dd-4d4d-87dd-de633bb3c67a)

**Comment by ihor-drachuk** [comment]
> /duplicate of https://github.com/microsoft/vscode-copilot/issues/11125

I can't access that link. Is it alive?

**Comment by jrieken** [comment]
released as 0.23.1
